src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.addEventListener('click', (e) => {
+  const heading = e.target.closest('th');
```

Comment:
No need add lestener to whole table, add just for table head

Reply:
done